staging/src/k8s.io/apiserver/plugin/pkg/authenticator/token/webhook/webhook.go

```diff
@@ -105,11 +106,13 @@ func (w *WebhookTokenAuthenticator) AuthenticateToken(token string) (user.Info,
 		}
 	}
 
```

Comment:
need to verify r.Status.Audience is one of the audiences in `auds` (we are a token review client here and must do the verification described in the TokenReview API doc)